helm/bits/templates/bits.yaml

```
@@ -65,10 +65,16 @@ spec:
         - name: bits-assets
           mountPath: /assets/
         {{ end }}
+        {{- if or .Values.global.resources.cpu .Values.global.resources.memory }}
         resources:
           requests:
-            cpu: 800m
-            memory: 150Mi
+          {{- if .Values.global.resources.cpu }}
+            cpu: {{ .Values.global.resources.cpu }}
+          {{ end }}
+          {{- if .Values.global.resources.memory }}
+            memory: {{ .Values.global.resources.memory }}
+          {{- end }}
```

Actually, doesn't this end up being:
```suggestion
          requests: {{ .Values.global.resources | toJson }}
```

---

Hm. That may actually be. I will take this back to `In Progress` and trial the simplification.

---

No, it does not work, not fully.

For compat I am keeping the old defaults here (800m, 150Mi), if in a different location (values yaml).

That means that for `kubecf` I have to allow an assignment of the form `memory: ~` to override these defaults to nil/nothing/undef.

With the `toJson` setup I get an error for that, namely:
> Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: unknown object type "nil" in Deployment.spec.template.spec.containers[0].resources.requests.memory

And simply leaving that assignment out gives me the default value, as to be expected.

Going back to the old form now, and using only `with` for some simplification.

---

Done with commit 935c4a4f960b8b455d6c76fb727c1a65cab05f6d.